content/agent/kubernetes/daemonset_setup.md

```diff
@@ -0,0 +1,305 @@
+            mountPath: /var/run/docker.sock
+          - name: procdir
+            mountPath: /host/proc
+            readOnly: true
+          - name: cgroups
+            mountPath: /host/sys/fs/cgroup
+            readOnly: true
+        livenessProbe:
+          exec:
+            command:
+            - ./probe.sh
+          initialDelaySeconds: 15
+          periodSeconds: 5
+      volumes:
+        - hostPath:
+            path: /var/run/docker.sock
+          name: dockersocket
+        - hostPath:
+            path: /proc
+          name: procdir
+        - hostPath:
+            path: /sys/fs/cgroup
+          name: cgroups
+```
+
+Replace `YOUR_API_KEY` with [your API key][5] or use [Kubernetes secrets][6] to set your API key [as an environment variable][7]. [Consult our docker integration to discover all configuration options.][8]
+
+Deploy the DaemonSet with the command:
+```
+kubectl create -f datadog-agent.yaml
+```
+
+**Note**:  This manifest enables Autodiscovery's auto configuration feature. To learn how to configure Autodiscovery, please refer to [its documentation][9].
+
+### Verification
+
+To verify the Datadog Agent is running in your environment as a DaemonSet, execute:
+
+    kubectl get daemonset
+
+If the Agent is deployed, you will see output similar to the text below, where `DESIRED` and `CURRENT` are equal to the number of nodes running in your cluster.
+
+    NAME            DESIRED   CURRENT   READY     UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+    datadog-agent   2         2         2         2            2           <none>          16h
+
+
+## Enable capabilities
+
+### Log Collection
+
+To enable [Log collection][10] with your DaemonSet:
+
+1. Set the `DD_LOGS_ENABLED` and `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` variable to true in your *env* section:
+
+    ```
+    (...)
+      env:
+        (...)
+        - name: DD_LOGS_ENABLED
+            value: "true"
+        - name: DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL
+            value: "true"
+    (...)
+    ```
+
+2. Mount the `pointdir` volume in *volumeMounts*:
+
+    ```
+      (...)
+        volumeMounts:
+          (...)
+          - name: pointerdir
+              mountPath: /opt/datadog-agent/run
+      (...)
+      volumes:
+        (...)
+        - hostPath:
+              path: /opt/datadog-agent/run
+            name: pointerdir
+      (...)
+    ```
+
+Learn more about this in [the Docker log collection documentation][11].
+
+### Trace Collection
+
+To enable [Trace collection][20] with your DaemonSet:
+
+1. Set the `DD_APM_ENABLED` variable to true in your *env* section:
+
+    ```
+    (...)
+      env:
+        (...)
+        - name: DD_APM_ENABLED
+            value: "true"
+    (...)
+    ```
+
+2. Uncomment the `# hostPort: 8126` line. 
+  This exposes the Datadog Agent tracing port on each of your Kubernetes nodes. 
+
+    **Warning**: This opens a port on your host. Make sure your firewall covers that correctly. 
```

Comment:
"covers that correctly" comes across as being really vague. Let's be bit more explicit that users should ensure their firewall only allows access from their applications or trusted sources.